packages/office-addin-dev-settings/package-lock.json

```diff
@@ -1,2810 +1,8 @@
 {
   "name": "office-addin-dev-settings",
   "version": "1.8.16",
-  "lockfileVersion": 2,
```

Comment:
Please use the latest npm. `npm install -g npm`. npm v7 now writes lock file version 2.